src/hypergol/cli/create_tensorflow_model.py

```diff
@@ -0,0 +1,87 @@
+from pathlib import Path
+import fire
+
+from hypergol.name_string import NameString
+from hypergol.hypergol_project import HypergolProject
+
+
+def create_model_block(modelBlockName, project):
+    content = project.render(
```

Comment:
This can probably be inlined
```
return project.render(...
```

Reply:
also it seems a bit weird you made this into a function but not the other renders below? Feel like this function could just be inlined